Schematic for censoring

seabbs opened the pull request:
Not sure this usefully improves things and I ran out of arrow making energy before wrapping up.

parksw3 commented:
* I don't think it's a good idea to use the same visuals (e.g., arrows) for truncation and censoring
* I tried to use arrows for directly observed delays
* Dashed lines for unobserved delays (or "not-yet-observed delays"). I think the latter naming is clearer, but maybe a bit clunky
* Using vertical bars for censoring
* Not sure what you were trying to convey with subject 2 in censoring so left it blank. But I think the red point (true event time) shouldn't be the end point? So put a left bound and do something on the right? I think that's the message you were trying to convey but not sure.

parksw3 reviewed (commented):
Made some more tweaks.

* I don't think it's a good idea to use the same visuals (e.g., arrows) for truncation and censoring
* I tried to use arrows for directly observed delays
* Dashed lines for unobserved delays (or "not-yet-observed delays"). I think the latter naming is clearer, but maybe a bit clunky
* Using vertical bars for censoring
* Not sure what you were trying to convey with subject 2 in censoring so left it blank. But I think the red point (true event time) shouldn't be the end point? So put a left bound and do something on the right? I think that's the message you were trying to convey but not sure.

seabbs commented:
Nice. 

- I agree on using bars vs arrows (I just had no idea how to do this). 
- I like the dashed/non-dashed lines. I changed to truncated delay as that is obviously exactly what it is but perhaps it is a bit on the nose. 
- For the other subjects I have tried to include a sample of the other kinds of censoring we might see and so probably want to discuss.. So: 
   - Wider than a day censoring
   - No censoring for primary event
   - No censoring for secondary event
- To make this more schematic like I've dropped the y axis line but see what you think. 

We might also want to think about including the interaction between censoring and truncation on this schematic (i.e having a observation time on B and having the censoring window overlap with it). 

I've merged for now but not implying this is final.
